src/osmviz/manager.py

```diff
@@ -257,8 +293,11 @@ def getTileURL(self, tile_coord, zoom):
         Given x, y coord of the tile to download, and the zoom level,
         returns the URL from which to download the image.
         """
-        params = (zoom, tile_coord[0], tile_coord[1])
-        return self.server+"/%d/%d/%d.png" % params
+        return self.url \
+            .replace("{z}", str(zoom)) \
+            .replace("{x}", str(tile_coord[0])) \
+            .replace("{y}", str(tile_coord[1])) \
+            .replace("{s}", str(self.scale))
```

Comment:
I think I know what went wrong. Turns out I was testing something different. Not replacing all placeholders gives this error.

```.python
Python 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:24:40) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> "{a}".format(a="123", b="345")
'123'
>>> "{a}".format(b="345")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'a'
>>>
```

Just to be sure I've tested the above using "Python 3.5.2 (v3.5.2:4def2a2901a5, Jun 25 2016, 22:18:55) [MSC v.1900 64 bit (AMD64)] on win32", same result. So I'll edit the rest tonight when I get home from work.